Assets/Scripts/Models/DialogBoxInformation/DialogBoxLuaInformation.cs

```diff
@@ -0,0 +1,29 @@
+﻿#region License
+// ====================================================
+// Project Porcupine Copyright(C) 2016 Team Porcupine
+// This program comes with ABSOLUTELY NO WARRANTY; This is free software, 
+// and you are welcome to redistribute it under certain conditions; See 
+// file LICENSE, which is part of this source code package, for details.
+// ====================================================
+#endregion
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Xml.Serialization;
+using UnityEngine;
+
+public class DialogBoxLuaInformation
```

Comment:
Don't name the classes with Lua. The implementation shouldn't be part of the name. Same goes for all the other names that have Lua, like the prefab, and the DialogBoxLua.